kibana/Dockerfile

```
@@ -7,8 +7,8 @@ FROM docker.elastic.co/kibana/kibana-oss:${ELK_VERSION}
 # https://docs.search-guard.com/latest/search-guard-versions
 # https://search.maven.org/search?q=g:com.floragunn%20AND%20a:search-guard-kibana-plugin&core=gav
 ARG ELK_VERSION
-ENV SG_VERSION=${ELK_VERSION}-14
-RUN kibana-plugin install https://repo1.maven.org/maven2/com/floragunn/search-guard-kibana-plugin/${SG_VERSION}/search-guard-kibana-plugin-${SG_VERSION}.zip
+ARG SG_VERSION_KIBANA
```

```suggestion
ARG SG_VERSION_KIBANA
ENV SG_VERSION_KIBANA=${SG_VERSION_KIBANA}
```
As ugly as it may seem this extra `ENV` is [necessary|https://docs.docker.com/engine/reference/builder/#using-arg-variables] in order to be able to use a build argument inside a `RUN` instruction.